Source/Scene/ModelExperimental/RenderResources.js

```diff
@@ -121,6 +121,47 @@ function NodeRenderResources(modelRenderResources, sceneNode) {
    * @private
    */
   this.attributes = [];
+
+  /**
+   * The index to give to the next vertex attribute added to the attributes array. POSITION
+   * takes the 0 index.
```

Comment:
minor wording tweak for clarity. Update this below in the primitive resources too

```suggestion
   * takes index 0.
```